configs/test/bot/setup/android.bash

```diff
@@ -13,6 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function run_bot () {
+  bot_directory=$INSTALL_DIRECTORY/bots/$(echo $1 | sed s/:/-/)
+  mkdir -p $bot_directory
+  cp -r $INSTALL_DIRECTORY/clusterfuzz $bot_directory/clusterfuzz
+  echo "Created bot directory $bot_directory."
+
+  while true; do
+    if ! $("$ADB_PATH/adb" -s "$1" wait-for-device); then
```

Comment:
ADB_PATH looks wrong initialized, it should be resources/platform/android dir of clusterfuzz.

Reply:
Good point. There's no reason to download platform tools in this script because of that, so that simplifies a few things. Done.